### ISSUE TITLE
Swiper triggered on keyboard suggested text clicked on IOS

### ISSUE THREAD

**Issue by azsidhu** [leecade/react-native-swiper#972]
### Which OS ?
IOS

### Version
Which versions are you using:
"react": "16.5.2",

- react-native-swiper 1.5.13?
- react-native v0.57.7

### Expected behaviour
When entered keyboard suggested text, it should enter the text into a text filed currently active.

### Actual behaviour
When entered keyboard suggested text, it swipes to 2nd tab

### How to reproduce it>
Apply two swipe views in swiper, click on keyboard entered text
-

**Comment by retyui** [comment]
This repo dead: https://github.com/leecade/react-native-swiper/issues/713
Your issue is like other 480+ issues, will not be resolved in the short term.

**Comment by fabriziobertoglio1987** [comment]
> This repo dead: #713
> Your issue is like other 480+ issues, will not be resolved in the short term.

I don't believe that it is worth maintaining .. we can build the functionality with [`ScrollView`](https://github.com/spencercarli/react-native-snapchat-clone/blob/master/app/components/Menu.js#L74)
